open-source-hosts-plugin/package.json

```diff
@@ -31,7 +31,7 @@
         "@types/jest": "^27.4.1",
         "@typescript-eslint/eslint-plugin": "^5.16.0",
         "@typescript-eslint/parser": "^5.16.0",
-        "babylonjs-editor": "^4.1.0",
+        "babylonjs-editor": "^4.1.1",
```

Comment:
Why do we directly use the newest v4.2.0?

Reply:
My understanding of versioning here is that we're specifying the _minimum_ version our plugin works with. Someone modifying the plugin for their own use could `npm install babylonjs-editor@4.2.0` and have the codebase still work, but someone could not do the same for `4.2.0` because we're indicating here that we don't support it.

In other words, we're not exclusively using anything from version 4.2.0 that would make us non-backwards compatible with 4.1.1